.github/ISSUE_TEMPLATE/module_proposal_bicep_child.yml

```diff
@@ -0,0 +1,228 @@
+name: New Bicep Child Module Proposal 🗒️
+description: Want a new child module of a Bicep resource module to exist? Let us know!
+title: "[Bicep Child Module Proposal]: `BICEP_CHILD_MODULE_NAME_BETWEEN_BACKTICKS`"
+labels: ["Type: New Module Proposal :bulb:", "Needs: Triage :mag:", "Language: Bicep :muscle:", "Class: Child Module :package:"]
+projects: ["Azure/529"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for proposing a new Bicep child module! Please create a separate proposal for each child module.
```

Comment:
```suggestion
        Thank you for requesting a new Bicep child module to be published in the public bicep registry! Please create a separate proposal for each child module.
```

Reply:
Suggestion mostly accepted: I changed the word requesting to proposing. We use this wording consistently, as the "requesting" may imply that it's granted it will be taken care of by someone exactly the way it's written. This may or may not be the case eventually.

Reply:
QQ: in line 2, you've changed the language from "new child module" to "child module". Should we be consistent here too?

Reply:
Agree, probably better to remove `new`

Reply:
Cool, I already updated it when I left the previous message. :)